code/game/objects/machinery/nuclearbomb.dm

```diff
@@ -92,6 +93,14 @@
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_NUKE_EXPLODED, z)
 	return TRUE
 
+/// Permanently disables this nuke, for use on hijack
+/obj/machinery/nuclearbomb/proc/disable_on_hijack()
+	desc += " Something seems to be wrong with it."
```

Comment:
```suggestion
	desc += "A strong interference renders this inoperable."
```

Reply:
Sounds good, but want to keep the leading space.
```suggestion
	desc += " A strong interference renders this inoperable."
```